gnomad_constraint/utils/constraint_basics.py

```diff
@@ -16,6 +17,8 @@
     filter_vep_transcript_csqs,
 )
 
+POPS = ("global", "afr", "amr", "eas", "nfe", "sas")
```

Comment:
it would be helpful to add some documentation about this constant. also, is there a reason why this constant needs to exist in two locations (this script and `gnomad_constraint/resources/resource_utils.py`)?